generate/syntax/main.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	sourceFilePath         = "kubedog.go"
+	destinationFilePath    = "docs/syntax.md"
+	newLine                = "\n"
+	actionIndicator        = "//syntax-generation"
+	actionDelimiter        = ":"
+	actionBegin            = "begin"
+	actionEnd              = "end"
+	actionTittle           = "tittle"
+	tittleBeginning        = "## "
```

Comment:
Should this be title? 

```suggestion
	titleBeginning        = "## "
```

Reply:
Thanks!